test/e2e/mobile/events.spec.js

```diff
@@ -58,6 +58,30 @@ describe('Events', () => {
     expect(onCellDblClick).toHaveBeenCalled();
   });
 
+  it('should preventDefault the touchend event (double-tap issue #7824)', async() => {
+    const hot = handsontable({
+      width: 400,
+      height: 400,
+    });
+
+    const cell = hot.getCell(1, 1);
+    const event = triggerTouchEvent('touchend', cell);
+
+    expect(event.defaultPrevented).toBeTrue();
+  });
+
+  it('should not preventDefault the touchend event when interactive element is clicked', async() => {
+    const hot = handsontable({
+      width: 400,
+      height: 400,
+    });
+
+    const cell = hot.getCell(1, 1);
+    const event = triggerTouchEvent('touchend', cell);
+
+    expect(event.defaultPrevented).toBeTrue();
+  });
+
```

Comment:
Am I missing something or this test doesn't use interactive elements and the assertion is `true` but should be `false`?

Reply:
Ah. I'll check that.

Reply:
I've precise the tests within https://github.com/handsontable/handsontable/pull/7824/commits/02a4f5aabede847233734a5dbd9dfad273615809.